go/sqltypes/query.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sqltypes
+
+import (
+	"bytes"
+)
+
+// EscapeID produces a backticked identifier given an input string.
+func EscapeID(in string) string {
+	var buf bytes.Buffer
+	WriteEscapeID(&buf, in)
+	return buf.String()
+}
+
+// WriteEscapeID produces a backticked identifier given an input string.
```

Comment:
The comment is almost the same.

Maybe s/produces/returns/ for `EscapeID` above and here say that its written to the buffer "buf"?

Reply:
Done